deploy/Mac_ARM-Delivery.yml

```diff
@@ -58,6 +58,15 @@ macOS_task:
     .ci/ci_controller.sh "$TARGET_ARCH" TEST
 
   package_script: |
+    otool -L run/john
+    if false; then
```

Comment:
After adding the libraries to the compressed file, there are two possibilities:

- either a simple `run/john` should work
- or one need to use this for it to work:
```sh
export DYLD_LIBRARY_PATH=.:$DYLD_LIBRARY_PATH
run/john
``` 

    DYLD_LIBRARY_PATH
    
    This is a colon separated list of directories that contain libraries. The dynamic linker searches these directories before it searches the default locations for libraries. 

My VM isn't clean, so I can't test it properly. But the below works for me:

```sh
export DYLD_LIBRARY_PATH=.
run/john --list=build-info
```

See also `DYLD_FALLBACK_LIBRARY_PATH`.